mappings/net/minecraft/world/PortalForcer.mapping

```diff
@@ -1,13 +1,32 @@
 CLASS net/minecraft/unmapped/C_daetwwvm net/minecraft/world/PortalForcer
+	FIELD f_bfoygtnv PORTAL_SPAWN_RADIUS_NETHER I
+	FIELD f_emmqbhvf PORTAL_SPAWN_RADIUS_OVERWORLD I
 	FIELD f_rcqxkdpj world Lnet/minecraft/unmapped/C_bdwnwhiu;
 	METHOD <init> (Lnet/minecraft/unmapped/C_bdwnwhiu;)V
 		ARG 1 world
+	METHOD m_atsnrvoq (Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_xhhbfljz;)D
+		ARG 1 pointOfInterest
+	METHOD m_hsbqvbno (Lnet/minecraft/unmapped/C_pneibfez;Lnet/minecraft/unmapped/C_xhhbfljz;)Z
+		ARG 1 pointOfInterest
+	METHOD m_kliyqyna (Lnet/minecraft/unmapped/C_xhhbfljz;)Z
+		ARG 1 pointOfInterest
```

Comment:
I think we use `poi` instead of `pointOfInterest`, but I might be wrong

Reply:
we do